js/main.js

```diff
@@ -0,0 +1,59 @@
+const hamburger = document.querySelector('.hamburger-container');
+const navbar = document.querySelector('.nav-list-items');
+
+hamburger.addEventListener(('click'), () => {
+  hamburger.classList.toggle('active');
+  navbar.classList.toggle('action');
+});
+
+// Array of project objects
+const seeMore = document.querySelector('.seemore');
+const seeInstructor = document.querySelector('.instructor-section');
+const instructorSection = document.querySelector('.instructor-container');
+
+seeMore.addEventListener(('click'), () => {
+  seeInstructor.classList.toggle('setoff');
+});
+
+const instructorContainer = [
+  {
+    name: 'Joseph Gonzalez',
+    title: 'Leading open-source projects at GitHub',
+    description: 'Joseph has been leading open-source projects at GitHub. Mostly related to JavaScript',
+    Image: 'img/joseph.jpg',
+  },
+  {
+    name: 'Jurica Koletic',
+    title: 'CEO of the marketing platform HubSpot',
+    description: 'Jurica is CEO of the marketing platform HubSpot, which he co-founded in 2006. ',
+    Image: 'img/jurica.jpg',
+  },
+  {
+    name: 'Sara Dam',
+    title: 'Fine artist-turned-coder',
+    description: 'Fine artist-turned-coder, Sara Dam took a total immersion approach to learning web development.',
+    Image: 'img/sara.jpg',
+  },
+  {
+    name: 'Ayo Ogunseinde',
+    title: 'CEO of Smartsheet',
+    description: 'Ayo the CEO of Smartsheet, a software-as-service (SaaS) company that offers applications for collaboration.',
+    Image: 'img/ayo.jpg',
+  },
+];
```

Comment:
- [ ] As required for the project and in order to match the Behance template design, I think it would be better if you add **two** other speakers on the speaker's list so your page will have **2** columns of **three** speakers each.

- [ ] To stick to the Behance template design, kindly make sure you also add the background image on each `instructor or speaker` photo like in the image below.

### Template Design
![Screenshot from 2022-09-22 12-45-12](https://user-images.githubusercontent.com/83042742/191727298-5c237bed-1ff4-4718-8075-e470aa9c9ce9.png)

### Your Design
![Screenshot from 2022-09-22 12-47-30](https://user-images.githubusercontent.com/83042742/191727532-22735ec6-5eed-42b5-adc1-97e968b08595.png)